routes/item/[id].tsx

```diff
@@ -112,7 +112,7 @@ export const handler: Handlers<ItemPageData, State> = {
     };
     await createNotification(notification);
 
-    return redirect(`/item/${ctx.params.id}`);
+    return redirect(`/`);
```

Comment:
Why was this done? Am I missing something?

Reply:
Probably just some fat finger on my side. Just fixed it now.